### ISSUE TITLE
API data is inconsistent.

### ISSUE THREAD

**Issue by FrederLu** [Magickbase/ckb-explorer-public-issues#715]
Request URL:   https://ckb-explorer-api-staging.magickbase.com/api/v2/bitcoin_addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf/rgb_cells?page=1&page_size=10&sort=capacity.desc

```json
{
                    "data": {
                        "id": "175291928",
                        "type": "cell_output",
                        "attributes": {
                            "cell_type": "xudt",
                            "tx_hash": "0xd5bb268266c1957c97c657943247f271a80246c88a99978e1db57cf485bbb67f",
                            "cell_index": 0,
                            "type_hash": "0x94d1b2046013c8b284c8714a10e0daf2db38b5db14442a0c81afb15967e239a7",
                            "data": "0x00e40b54020000000000000000000000",
                            "block_number": "13635232",
                            "capacity": "25499993990.0",
                            "occupied_capacity": "15900000000",
                            "block_timestamp": "1718785502382",
                            "type_script": {
                                "args": "0x78f569b97d6a984edd734f3304bf3581d45b5be7338ebb7d840247aab72b896020",
                                "code_hash": "0x25c29dc317811a6f6f3985a7a9ebc4838bd388d19d0feeecf0bcd60f6c0975bb",
                                "hash_type": "type"
                            },
                            "lock_script": {
                                "args": "0x01000000344ea31a6bd6154c5bb4ddb4ca4b63cccbda9c42739f4034c7ec675e10d8a7b2",
                                "code_hash": "0xd07598deec7ce7b5665310386b4abd06a6d48843e953c5cc2112ad0d5a220364",
                                "hash_type": "type"
                            },
                            "extra_info": {
                                "symbol": "XTT20",
                                "amount": "10000000000",
                                "decimal": "8",
                                "type_hash": "0x94d1b2046013c8b284c8714a10e0daf2db38b5db14442a0c81afb15967e239a7",
                                "published": true,
                                "type": "xudt"
                            }
                        }
                    }
                }

{
                    "data": {
                        "id": "175913626",
                        "type": "cell_output",
                        "attributes": {
                            "cell_type": "xudt",
                            "tx_hash": "0x07828bb2198a06c053db430122b8cb551ab6caa1bb3a42a681994feac49db83e",
                            "cell_index": 0,
                            "type_hash": "0x94d1b2046013c8b284c8714a10e0daf2db38b5db14442a0c81afb15967e239a7",
                            "data": "0x00e40b54020000000000000000000000",
                            "block_number": "13687177",
                            "capacity": "25499993990.0",
                            "occupied_capacity": "15900000000",
                            "block_timestamp": "1719201440669",
                            "type_script": {
                                "args": "0x78f569b97d6a984edd734f3304bf3581d45b5be7338ebb7d840247aab72b896020",
                                "code_hash": "0x25c29dc317811a6f6f3985a7a9ebc4838bd388d19d0feeecf0bcd60f6c0975bb",
                                "hash_type": "type"
                            },
                            "lock_script": {
                                "args": "0x010000007a838271eb66cd4bed61af1dd20426054e1448fdce7e55b451ab309289d66598",
                                "code_hash": "0xd07598deec7ce7b5665310386b4abd06a6d48843e953c5cc2112ad0d5a220364",
                                "hash_type": "type"
                            },
                            "extra_info": {
                                "symbol": "XTT20",
                                "amount": "10000000000",
                                "decimal": "8",
                                "type_hash": "0x94d1b2046013c8b284c8714a10e0daf2db38b5db14442a0c81afb15967e239a7",
                                "published": true,
                                "type": "xudt"
                            }
                        }
                    }
                }

{
                    "data": {
                        "id": "175951179",
                        "type": "cell_output",
                        "attributes": {
                            "cell_type": "xudt",
                            "tx_hash": "0x1be3ce2a212afed4e95e79359324293893ba0d8a9aeafe7c22829ce1afb77a3f",
                            "cell_index": 0,
                            "type_hash": "0x94d1b2046013c8b284c8714a10e0daf2db38b5db14442a0c81afb15967e239a7",
                            "data": "0x00c5015a020000000000000000000000",
                            "block_number": "13690911",
                            "capacity": "15900000000.0",
                            "occupied_capacity": "15900000000",
                            "block_timestamp": "1719231660782",
                            "type_script": {
                                "args": "0x78f569b97d6a984edd734f3304bf3581d45b5be7338ebb7d840247aab72b896020",
                                "code_hash": "0x25c29dc317811a6f6f3985a7a9ebc4838bd388d19d0feeecf0bcd60f6c0975bb",
                                "hash_type": "type"
                            },
                            "lock_script": {
                                "args": "0x010000007a838271eb66cd4bed61af1dd20426054e1448fdce7e55b451ab309289d66598",
                                "code_hash": "0xd07598deec7ce7b5665310386b4abd06a6d48843e953c5cc2112ad0d5a220364",
                                "hash_type": "type"
                            },
                            "extra_info": {
                                "symbol": "XTT20",
                                "amount": "10100000000",
                                "decimal": "8",
                                "type_hash": "0x94d1b2046013c8b284c8714a10e0daf2db38b5db14442a0c81afb15967e239a7",
                                "published": true,
                                "type": "xudt"
                            }
                        }
                    }
                }
```

---

Request URL: https://ckb-explorer-api-staging.magickbase.com/api/v1/addresses/tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf

```json
{
            "id": "25434985",
            "type": "address",
            "attributes": {
                "lock_info": null,
                "address_hash": "ckt1qrg8txx7a37w0dtx2vgrs662h5r2d4ygg05483wvyyf26r26ygpkgqgpqqqqqdzw5vdxh4s4f3dmfhd5ef9k8nxtm2wyyuulgq6v0mr8tcgd3fajx6lpaz",
                "balance": "25499993990",
                "transactions_count": "1",
                "dao_deposit": "0",
                "interest": "0",
                "is_special": "false",
                "live_cells_count": "1",
                "mined_blocks_count": "0",
                "average_deposit_time": "",
                "udt_accounts": [
                    {
                        "symbol": "XTT20",
                        "decimal": "8",
                        "amount": "10000000000",
                        "type_hash": "0x94d1b2046013c8b284c8714a10e0daf2db38b5db14442a0c81afb15967e239a7",
                        "udt_icon_file": null,
                        "udt_type": "xudt",
                        "udt_type_script": {
                            "args": "0x78f569b97d6a984edd734f3304bf3581d45b5be7338ebb7d840247aab72b896020",
                            "code_hash": "0x25c29dc317811a6f6f3985a7a9ebc4838bd388d19d0feeecf0bcd60f6c0975bb",
                            "hash_type": "type"
                        }
                    }
                ],
                "lock_script": {
                    "args": "0x01000000344ea31a6bd6154c5bb4ddb4ca4b63cccbda9c42739f4034c7ec675e10d8a7b2",
                    "code_hash": "0xd07598deec7ce7b5665310386b4abd06a6d48843e953c5cc2112ad0d5a220364",
                    "hash_type": "type"
                },
                "dao_compensation": "0",
                "balance_occupied": "25499993990",
                "bitcoin_address_hash": "tb1qdj3cv8jktgvl4ey4h7v24xuthc8ta99gj4a6pf"
            }
        },
        {
            "id": "25488675",
            "type": "address",
            "attributes": {
                "lock_info": null,
                "address_hash": "ckt1qrg8txx7a37w0dtx2vgrs662h5r2d4ygg05483wvyyf26r26ygpkgqgpqqqqq75rsfc7kekdf0kkrtca6gzzvp2wz3y0mnn72k69r2esj2yavevc0z94tz",
                "balance": "57700000000",
                "transactions_count": "6",
                "dao_deposit": "0",
                "interest": "0",
                "is_special": "false",
                "live_cells_count": "4",
                "mined_blocks_count": "0",
                "average_deposit_time": "",
                "udt_accounts": [
                    {
                        "symbol": "XTT20",
                        "decimal": "8",
                        "amount": "10100000000",
                        "type_hash": "0x94d1b2046013c8b284c8714a10e0daf2db38b5db14442a0c81afb15967e239a7",
                        "udt_icon_file": null,
                        "udt_type": "xudt",
                        "udt_type_script": {
                            "args": "0x78f569b97d6a984edd734f3304bf3581d45b5be7338ebb7d840247aab72b896020",
                            "code_hash": "0x25c29dc317811a6f6f3985a7a9ebc4838bd388d19d0feeecf0bcd60f6c0975bb",
                            "hash_type": "type"
                        }
                    },
                    {
                        "symbol": "XTT21",
                        "decimal": "8",
                        "amount": "10000000000",
                        "type_hash": "0x0fd1864cde446e0fc0d2c7028e7d968244ddcd1f035ea01c9344e7ec71323431",
                        "udt_icon_file": null,
                        "udt_type": "xudt",
                        "udt_type_script": {
                            "args": "0x556f5aaff56a316c64e42e93259e734f82d96dc3639ae061cb456e93d285f1d621",
                            "code_hash": "0x25c29dc317811a6f6f3985a7a9ebc4838bd388d19d0feeecf0bcd60f6c0975bb",
                            "hash_type": "type"
                        }
                    }
```

The data information about "type_hash": "0x94d1b2046013c8b284c8714a10e0daf2db38b5db14442a0c81afb15967e239a7" of 2 APIs is inconsistent.

**Comment by rabbitz** [comment]
fix pr: https://github.com/nervosnetwork/ckb-explorer/pull/2045

**Comment by FrederLu** [comment]
> fix pr: [nervosnetwork/ckb-explorer#2045](https://github.com/nervosnetwork/ckb-explorer/pull/2045)

Verified.
